third_party/py/dataclasses/BUILD

```diff
@@ -0,0 +1,19 @@
+# Description:
+#   dataclasses module from the Python 3.7 standard library.
+#   For backporting to Python 3.6 and below.
+
+package(default_visibility = ["//visibility:public"])
+
+licenses(["notice"])  # Python Software Foundation
+
+filegroup(
```

Comment:
I saw the CL explicitly adding this; but why?

Reply:
Fixes failing presubmit https://buildkite.com/bazel/bazel-bazel-github-presubmit/builds/6240#65bca58b-2680-422b-aabc-4ae9cc015e5e (https://storage.googleapis.com/bazel-untrusted-buildkite-artifacts/65bca58b-2680-422b-aabc-4ae9cc015e5e/src/test/shell/bazel/srcs_test/attempt_1.log).